osf_oauth2_adapter/migrations/0002_application.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2017-03-09 15:34
+from __future__ import unicode_literals
+
+import os
+
+from django.db import migrations
+
+
+def create_socialapp(apps, schema_editor):
+    Site = apps.get_model('sites', 'site')
+    SocialApp = apps.get_model('socialaccount', 'socialapp')
+    if not SocialApp.objects.exists():
+        app = SocialApp.objects.using(schema_editor.connection.alias).create(
+            key='',
+            name='OSF',
+            provider='osf',
+            # Defaults are valid for staging
+            client_id=os.environ.get('OSF_OAUTH_CLIENT_ID', '3518b74e12584abf9e48565ff6aee6f3'),
+            secret=os.environ.get('OSF_OAUTH_SECRET', 'vYlku3raTL5DnHZlkqCIaShmPVIl1nifsFJCNLxU'),
+        )
+        app.sites.add(Site.objects.get(domain='osf.io'))
+
+
+def create_site(apps, schema_editor):
+    Site = apps.get_model('sites', 'site')
+    if not Site.objects.filter(domain='osf.io').exists():
+        Site.objects.create(domain='osf.io', name='OSF')
```

Comment:
It's because our SITE_ID in defaults.py is `1`, which is example.com, because it gets created first. I'll change this migration to make a lookit site and change our SITE_ID to be `2`. That should fix this initial fail.